Gemfile

```diff
@@ -17,12 +17,13 @@ gem 'bootstrap_form'
 gem 'sidekiq'
 #gem 'sentry-raven'
 
+gem 'fog'
+gem 'fog-aws'
```

Comment:
Currently I am using local storage, I forgot to test AWS S3 :-)  Thanks for reminding me to test.